ceph.spec.in

```diff
@@ -291,13 +291,12 @@ BuildRequires:	xmlsec1-nss
 %endif
 BuildRequires:	xmlsec1-openssl
 BuildRequires:	xmlsec1-openssl-devel
+BuildRequires:	python%{python3_pkgversion}-cherrypy
 BuildRequires:	python%{python3_pkgversion}-jwt
+BuildRequires:	python%{python3_pkgversion}-routes
+BuildRequires:	python%{python3_pkgversion}-scipy
```

Comment:
Would it make sense (if it works) to move Requires and BuildRequires to subpackage scope? There's this recurrent question on which package requires _this package_, and so it would be easier to find.

Reply:
You mean in make_check sections?

It makes sense, but I'd prefer to address this in another PR.

Reply:
i concur with @epuertat . some of these packages are only required at runtime. and hence can moved to "make_check" *and* "dashboard" 's Requires.